fictionary/__init__.py

```diff
@@ -251,17 +282,12 @@ def main(argv=sys.argv[1:]):
                       " min-length!", file=sys.stderr)
                 return -1
 
-        LOG.setLevel(logging.DEBUG if args.verbose else logging.WARNING)
-
-        path = join(DATA_FILE_ROOT, 'dictionary.dat')
-        with DataFile(path, refresh=args.refresh) as shelf:
-            model = shelf[args.dictionary]
-            for _ in range(args.count):
-                real_word_filter = lambda w: not shelf.is_real_word(''.join(w))
-                print(''.join(
-                    model.random_sequence(
-                        args.min_length, args.max_length, real_word_filter)
-                ))
+        random_words = get_random_words(
```

Comment:
I love how clean this looks now!

Reply:
Please could you add some tests for the `get_random_words` function.

Reply:
Done, added tests.